core/state.go

```diff
@@ -150,25 +127,25 @@ func (s *state) getCommittedSeals() []*messages.CommittedSeal {
 	return s.seals
```

Comment:
I believe all of get* methods except of `getCommitSent` are not thread safe as far as they return reference types without deep-copying them.

Reply:
@Kourin1996

Reply:
Added deep copy c830856d3d236c70e222dcca8073d99da69aa522